plugins/katello/3.14/installation/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Katello Installation
+script: osmenu.js
+---
+
+# Katello {{ page.version }} Installation
+
+{% if page.version == 'nightly' %}
+  <div class="alert alert-danger">
+    These are the instructions for installing the unstable nightly release of Katello!
+  </div>
+{% elsif page.version != page.latest %}
+  <div class="alert alert-danger">
+    These instructions are for installing Katello {{ page.version }}, but the latest stable is <a href="/plugins/katello/{{ page.latest }}/installation/index.html">{{ page.latest }}</a>.
+  </div>
+{% endif %}
+
+Note: After installation of Katello, be sure to trust Katello's CA certificate
```

Comment:
After installation of Katello, navigate to the `/pub` directory and trust the Katello's CA certificate for identifying web cites (e.g. `http://katello.example.com/pub/katello-server-ca.crt`).

Reply:
Is the distinction here that you want to omit the 'note' part?

Reply:
No, sorry. My  intention was to suggest you replace all of the text after Note: with the text I have proposed. The import thing is to remove "be sure to trust" as its better to just tell epole what to do, and then the "You will find" is just too wordy. 

Adding "This is required for the encrypted NoVNC connections." is OK, but would be best left to a procedure on NoVNC as trusting the cert is also required for other reasons.

Reply:
@jturel I added the Note: back.

Reply:
Updated. I took the liberty of omitting 'Note' and surrounding the message with an 'info' CSS style to make appropriately bring attention to it